next-sitemap.config.js

```diff
@@ -4,9 +4,37 @@
  */
 
 /** @type {import('next-sitemap').IConfig} */
+const siteUrl = process.env.SITE_URL || 'https://developer.hashicorp.com'
+
 module.exports = {
-	siteUrl: process.env.SITE_URL || 'https://developer.hashicorp.com',
-	generateRobotsTxt: true, // (optional)
-	exclude: ['/swingset*', '/onboarding/*', '/profile*'],
-	// ...other options
+	siteUrl,
+	generateRobotsTxt: true,
+	robotsTxtOptions: {
+		additionalSitemaps: [
+			`${siteUrl}/hcp-sitemap.xml`,
+			`${siteUrl}/sitemap-docs.xml`,
+		],
+	},
+	exclude: [
+		'/swingset*',
+		'/onboarding/*',
+		'/profile*',
+		'sitemap.xml',
+		'/boundary/docs*',
+		'/boundary/commands*',
+		'/consul/docs*',
+		'/consul/commands*',
+		'/nomad/docs*',
+		'/nomad/commands*',
+		'/packer/docs*',
+		'/packer/commands*',
+		'/terraform/docs*',
+		'/terraform/commands*',
+		'/vagrant/docs*',
+		'/vagrant/commands*',
+		'/waypoint/docs*',
+		'/waypoint/commands*',
+		'/vault/docs*',
+		'/vault/commands*',
```

Comment:
These all have different base paths, you can see the data files in `src/data/{product}` for a list of applicable base docs paths (minus `downloads`)

Reply:
I should have known this. I went through and added the correct paths. Thanks for catching this.